src/main/kotlin/no/nav/familie/tilbake/kravgrunnlag/MottattKravgrunnlagTask.kt

```diff
@@ -0,0 +1,28 @@
+package no.nav.familie.tilbake.kravgrunnlag
+
+import no.nav.familie.prosessering.AsyncTaskStep
+import no.nav.familie.prosessering.TaskStepBeskrivelse
+import no.nav.familie.prosessering.domene.Task
+import no.nav.familie.tilbake.repository.tbd.Kravgrunnlag431Repository
+import org.slf4j.LoggerFactory
+import org.springframework.stereotype.Service
+
+@Service
+@TaskStepBeskrivelse(taskStepType = MottattKravgrunnlagTask.BEHANDLE_KRAVGRUNNLAG,
+    maxAntallFeil = 20,
+    beskrivelse = "Håndter mottatt kravgrunnlag fra oppdrag",
+    triggerTidVedFeilISekunder = 60 * 60 * 12)
+class MottattKravgrunnlagTask(private val kravgrunnlag431Repository: Kravgrunnlag431Repository) : AsyncTaskStep {
+
+    private val LOG = LoggerFactory.getLogger(MottattKravgrunnlagTask::class.java)
```

Comment:
LOG skulle vært med små bokstaver. 
Personlig så liker jeg å bruke this::class.java i stedet for statisk referanse. Koreter å skrive, og kan kopieres mellom klasser og fremdeles være riktig.

Reply:
👍  for kopierbar kode